### DISCUSSION TITLE
Expand custom User field value

### DISCUSSION THREAD

**Discussion by shchasny** [pnp/pnpcore#382] [Q&A]
Hello all,

I would like to get Title property from custom User field for particular item.
I've tried to get it that way
```csharp
var item = await context.Web.Lists.GetByTitle(listTitle).Items.GetById(itemId).GetAsync(i => i["Custom_Owner"]);
```
and that
```csharp
var item = await context.Web.Lists.GetByTitle(listTitle).Items.QueryProperties(i => i["Custom_Owner"]).ToListAsync();
```
and its variations with ```(i["Custom_Owner"] as IFieldUserValue).Title```. But as a result, I get only LookupId, the rest - "Property .. was not yet loaded".

Could you help, please?

Thank you!

**Comment by jansenbe** [comment]
Something like this should do it (on top of my head, did not run below code):

```csharp
var myList = context.Web.Lists.GetByTitle(listTitle, p => p.Title, p => p.Items, p => p.Fields.QueryProperties(p => p.InternalName, p => p.FieldTypeKind, p => p.TypeAsString, p => p.Title));
foreach(var addedItem in myList.Items.AsRequested())
{
  var title = addedItem["Custom_Owner"] as IFieldUserValue).Title;
}
```

**Reply by shchasny** [reply]
Nope, it doesn't work :(
![image](https://user-images.githubusercontent.com/20610579/113681466-aaeab200-96ca-11eb-9ee2-c95fe144af9a.png)

**Reply by jansenbe** [reply]
These properties are not always returned by the query to load the list item data I think: can you verify that the returned json contains these values? You can use for example Fiddler to see the network traffic.

You do have the user id (10), so you could consider using that to load the remaining user info? You can use context.Web.GetUserById() to do that

**Reply by shchasny** [reply]
1. Simple 'web/lists/getByTitle(listtitle)/items/getById(itemId)' request returns these values.
2. Of course I can use additional request to get it. But why can't(or how can) I use simple expand\select to get that or another lookup field?
